quant.admin.inc

```diff
@@ -424,11 +424,15 @@ function quant_seed_settings() {
     '#default_value' => variable_get('quant_custom_routes', FALSE),
   ];
 
-  $form['quant_robots'] = [
-    '#type' => 'checkbox',
-    '#title' => t('Export robots.txt'),
-    '#default_value' => variable_get('quant_robots', FALSE),
-  ];
+  // Only show if it exists.
+  $file = $_SERVER['DOCUMENT_ROOT'] . '/robots.txt';
```

Comment:
I don't think we can do this, how does this work if the site is using a dynamically generated robots.txt file? For example using robotstxt module this doesn't create the file and just configures a route in PHP.

Reply:
It probably needs to be a HTTP request to `/robots.txt` to see if the site is responding with it.

Reply:
@steveworley good catch! updated :)